src/store/store.js

```diff
@@ -19,9 +23,12 @@ const historyReducer = connectRouter(history)(rootReducer);
 const persistedReducer = persistReducer(persistConfig, historyReducer);
 
 const myRouterMiddleware = routerMiddleware(history);
-const middleware = [myRouterMiddleware];
+const sagaMiddleware = createSagaMiddleware();
+const middleware = [myRouterMiddleware, sagaMiddleware];
 
 const store = createStore(persistedReducer, composeWithDevTools(applyMiddleware(...middleware)));
 const persistor = persistStore(store);
 
+sagaMiddleware.run(authenticationSagas);
```

Comment:
In sagas.js i would just combine all the other sagas from different application components and just run a rootSaga here.